components/Input/input.css

```diff
@@ -0,0 +1,77 @@
+/* global css file for fonts and icons  */
+@import '/css/main.css';
+
+/* input class main*/
+.input-types {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    margin: 1rem;
+    padding: 1rem;
+    height: 20rem;
+    width: 30rem;
+    background-color: var(--background-universal-color);
+
+}
+/* global html5 input element */
+input {
+    width: 20rem;
+    height: 2rem;
+    margin: 0.3rem;
+    padding: 0.2rem;
+    border-radius: 0.4rem;
+    color: white;
+}
+/* html5 input type submit button */
+input[type='submit'] {
+    height: 2rem;
+    width: 5rem;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    cursor: pointer;
+    font-size: 1.2rem;
+    font-weight: bold;
+    color: white;
+   
+}
+
```

Comment:
Don't use global styling here, it would get you in problems while you use your CL